### PR TITLE
Color de-duplication #2

### PR DESCRIPTION
**Pull request by SimonBrandner**
Type: task

<hr>

For reviewer: this should be reviewable commit by commit

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://6124a17aa41ce29a2acf0372--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/structures/_LeftPanelWidget.scss [hunk 1]
```diff
@@ -62,7 +62,7 @@ limitations under the License.
                     mask-position: center;
                     mask-size: contain;
                     mask-repeat: no-repeat;
-                    background-color: $roomlist-header-color;
+                    background-color: $tertiary-content;
```
Comment (turt2live) [inline res/css/structures/_LeftPanelWidget.scss:65]: this is an unsafe change due to custom theme support

File: res/css/structures/_SpacePanel.scss [hunk 2]
```diff
@@ -46,7 +46,7 @@ $activeBorderColor: $secondary-fg-color;
         mask-repeat: no-repeat;
         margin-left: $gutterSize;
         margin-bottom: 12px;
-        background-color: $roomlist-header-color;
+        background-color: $tertiary-content;
```
Comment (turt2live) [inline res/css/structures/_SpacePanel.scss:49]: unsafe due to custom themes

File: res/css/structures/_SpacePanel.scss [hunk 3]
```diff
@@ -158,7 +158,7 @@ $activeBorderColor: $secondary-fg-color;
             mask-position: center;
             mask-size: 20px;
             mask-repeat: no-repeat;
-            background-color: $roomlist-header-color;
+            background-color: $tertiary-content;
```
Comment (turt2live) [inline res/css/structures/_SpacePanel.scss:161]: unsafe due to custom themes

File: res/css/structures/_UserMenu.scss [hunk 4]
```diff
@@ -35,7 +35,7 @@ limitations under the License.
         // we cheat opacity on the theme colour with an after selector here
         &::after {
             content: '';
-            border-bottom: 1px solid $primary-fg-color; // XXX: Variable abuse
+            border-bottom: 1px solid $primary-content; // XXX: Variable abuse
```
Comment (turt2live) [inline res/css/structures/_UserMenu.scss:38]: the comment can probably leave now

File: res/css/structures/_UserMenu.scss [hunk 5]
```diff
@@ -176,7 +176,7 @@ limitations under the License.
             width: 85%;
             opacity: 0.2;
             border: none;
-            border-bottom: 1px solid $primary-fg-color; // XXX: Variable abuse
+            border-bottom: 1px solid $primary-content; // XXX: Variable abuse
```
Comment (turt2live) [inline res/css/structures/_UserMenu.scss:179]: the comment can probably leave now

File: res/css/views/spaces/_SpaceCreateMenu.scss [hunk 6]
```diff
@@ -40,7 +40,8 @@ $spacePanelWidth: 71px;
 
             > p {
                 font-size: $font-15px;
-                color: $secondary-fg-color;
+                color: $secondary-content;
+                margin: 0;
```
Comment (turt2live) [inline res/css/views/spaces/_SpaceCreateMenu.scss:44]: this seems to have snuck in here - what issue is it fixing?

File: res/themes/light-custom/css/_custom.scss [hunk 7]
```diff
@@ -84,7 +84,7 @@ $authpage-lang-color: var(--timeline-text-color);
 $roomheader-color: var(--timeline-text-color);
 // --roomlist-text-secondary-color
 $roomtile-preview-color: var(--roomlist-text-secondary-color);
-$roomlist-header-color: var(--roomlist-text-secondary-color);
+$tertiary-content: var(--roomlist-text-secondary-color);
```
Comment (turt2live) [inline res/themes/light-custom/css/_custom.scss:87]: unsafe assumption

File: res/themes/light-custom/css/_custom.scss [hunk 8]
```diff
@@ -36,9 +36,9 @@ $button-secondary-bg-color: var(--timeline-background-color);
 $field-focused-label-bg-color: var(--timeline-background-color);
 $lightbox-border-color: var(--timeline-background-color);
 $menu-bg-color: var(--timeline-background-color);
-$avatar-bg-color: var(--timeline-background-color);
+$background: var(--timeline-background-color);
```
Comment (turt2live) [inline res/themes/light-custom/css/_custom.scss:39]: unsafe assumption

**Review by turt2live — COMMENTED** [review]
Otherwise seems sane; just looks like a couple custom theme issues to work out.

**Comment by SimonBrandner** [comment]
Blocking until RC to be sure as this is a big change

**Review by turt2live — APPROVED** [review]

**Pull request merged**